secret-squirrel.js

```diff
@@ -7,6 +7,9 @@ module.exports = {
 			'tools/x-docs/static/storybook',
 			'tools/x-docs/static/components',
 			'tools/x-docs/public-prod/x-dash',
+			'packages/x-engine/.npmignore',
+			'components/x-follow-button/.bowerrc',
+			'components/x-teaser/.npmignore',
```

Comment:
`.npmignore` is allowed now =]

Reply:
<img width="323" alt="screen shot 2018-07-31 at 11 34 08" src="https://user-images.githubusercontent.com/10063385/43454752-e62b95c4-94b5-11e8-8547-2da7524a804e.png">
Interesting, should be allowed, but getting this when try to delete them

Reply:
Must need the latest `secret-squirrel` (2.10.1)

Reply:
And it is there, that makes it even more confusing